{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1204, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1204, "_consider_importhook"]}
{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1204, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1204, "_consider_importhook"]}
{"pytest_version": "7.4.4", "$report_type": "SessionStart"}
{"nodeid": "tests27_56_5.py::test_presence_of_inserted_element", "location": ["tests27_56_5.py", 4, "test_presence_of_inserted_element"], "keywords": {"test_presence_of_inserted_element": 1, "tests27_56_5.py": 1, "Folder_56": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00013565004337579012, "start": 1731092426.0877328, "stop": 1731092426.087869, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "fc29ca3d780c47baae7d1c5c924dd24a", "node": "<WorkerController gw0>"}
{"nodeid": "tests27_56_5.py::test_presence_of_inserted_element_at_index_21", "location": ["tests27_56_5.py", 20, "test_presence_of_inserted_element_at_index_21"], "keywords": {"test_presence_of_inserted_element_at_index_21": 1, "tests27_56_5.py": 1, "Folder_56": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0001744530163705349, "start": 1731092426.0879128, "stop": 1731092426.0880878, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw2", "testrun_uid": "fc29ca3d780c47baae7d1c5c924dd24a", "node": "<WorkerController gw2>"}
{"nodeid": "tests27_56_5.py::test_index_of_inserted_element", "location": ["tests27_56_5.py", 10, "test_index_of_inserted_element"], "keywords": {"test_index_of_inserted_element": 1, "tests27_56_5.py": 1, "Folder_56": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0001915500033646822, "start": 1731092426.0880013, "stop": 1731092426.0881937, "$report_type": "TestReport", "item_index": 1, "worker_id": "gw1", "testrun_uid": "fc29ca3d780c47baae7d1c5c924dd24a", "node": "<WorkerController gw1>"}
{"nodeid": "tests27_56_5.py::test_presence_of_inserted_element_at_index_21", "location": ["tests27_56_5.py", 20, "test_presence_of_inserted_element_at_index_21"], "keywords": {"test_presence_of_inserted_element_at_index_21": 1, "tests27_56_5.py": 1, "Folder_56": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/Command_T_D/Q27/command_results_5/Folder_56/generated_answer.py", "lineno": 7, "message": "Failed: Timeout >7.0s"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_presence_of_inserted_element_at_index_21():", "        initial_list = [i for i in range((21 + 1) * 2)]", ">       returned_list = insert_after_index(initial_list)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q27/command_results_5/Folder_56/tests27_56_5.py", "lineno": 23, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def insert_after_index(nums):", "        ins_idx = 21", "        ins_val = 655.24", "        idx = sys.maxsize - 1", ">       while idx > ins_idx:", "E       Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["nums", "[0, 1, 2, 3, 4, 5, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q27/command_results_5/Folder_56/generated_answer.py", "lineno": 7, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_presence_of_inserted_element_at_index_21():", "        initial_list = [i for i in range((21 + 1) * 2)]", ">       returned_list = insert_after_index(initial_list)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q27/command_results_5/Folder_56/tests27_56_5.py", "lineno": 23, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def insert_after_index(nums):", "        ins_idx = 21", "        ins_val = 655.24", "        idx = sys.maxsize - 1", ">       while idx > ins_idx:", "E       Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["nums", "[0, 1, 2, 3, 4, 5, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q27/command_results_5/Folder_56/generated_answer.py", "lineno": 7, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/Command_T_D/Q27/command_results_5/Folder_56/generated_answer.py", "lineno": 7, "message": "Failed: Timeout >7.0s"}, null]]}, "when": "call", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (130371286992448) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 6.9994563210057095, "start": 1731092426.088469, "stop": 1731092433.087926, "extras": [], "$report_type": "TestReport", "item_index": 2, "worker_id": "gw2", "testrun_uid": "fc29ca3d780c47baae7d1c5c924dd24a", "node": "<WorkerController gw2>"}
{"nodeid": "tests27_56_5.py::test_index_of_inserted_element", "location": ["tests27_56_5.py", 10, "test_index_of_inserted_element"], "keywords": {"test_index_of_inserted_element": 1, "tests27_56_5.py": 1, "Folder_56": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/Command_T_D/Q27/command_results_5/Folder_56/generated_answer.py", "lineno": 7, "message": "Failed: Timeout >7.0s"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_index_of_inserted_element():", "        initial_list = [i for i in range((21 + 1) * 2)]", ">       returned_list = insert_after_index(initial_list)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q27/command_results_5/Folder_56/tests27_56_5.py", "lineno": 13, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def insert_after_index(nums):", "        ins_idx = 21", "        ins_val = 655.24", "        idx = sys.maxsize - 1", ">       while idx > ins_idx:", "E       Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["nums", "[0, 1, 2, 3, 4, 5, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q27/command_results_5/Folder_56/generated_answer.py", "lineno": 7, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_index_of_inserted_element():", "        initial_list = [i for i in range((21 + 1) * 2)]", ">       returned_list = insert_after_index(initial_list)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q27/command_results_5/Folder_56/tests27_56_5.py", "lineno": 13, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def insert_after_index(nums):", "        ins_idx = 21", "        ins_val = 655.24", "        idx = sys.maxsize - 1", ">       while idx > ins_idx:", "E       Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["nums", "[0, 1, 2, 3, 4, 5, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q27/command_results_5/Folder_56/generated_answer.py", "lineno": 7, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/Command_T_D/Q27/command_results_5/Folder_56/generated_answer.py", "lineno": 7, "message": "Failed: Timeout >7.0s"}, null]]}, "when": "call", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (137681587668544) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 6.999323535012081, "start": 1731092426.0886002, "stop": 1731092433.0879245, "extras": [], "$report_type": "TestReport", "item_index": 1, "worker_id": "gw1", "testrun_uid": "fc29ca3d780c47baae7d1c5c924dd24a", "node": "<WorkerController gw1>"}
{"nodeid": "tests27_56_5.py::test_index_of_inserted_element", "location": ["tests27_56_5.py", 10, "test_index_of_inserted_element"], "keywords": {"test_index_of_inserted_element": 1, "tests27_56_5.py": 1, "Folder_56": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (137681587668544) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 0.00013270298950374126, "start": 1731092433.0973604, "stop": 1731092433.0974936, "$report_type": "TestReport", "item_index": 1, "worker_id": "gw1", "testrun_uid": "fc29ca3d780c47baae7d1c5c924dd24a", "node": "<WorkerController gw1>"}
{"nodeid": "tests27_56_5.py::test_presence_of_inserted_element_at_index_21", "location": ["tests27_56_5.py", 20, "test_presence_of_inserted_element_at_index_21"], "keywords": {"test_presence_of_inserted_element_at_index_21": 1, "tests27_56_5.py": 1, "Folder_56": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (130371286992448) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 0.00011593196541070938, "start": 1731092433.097172, "stop": 1731092433.0972888, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw2", "testrun_uid": "fc29ca3d780c47baae7d1c5c924dd24a", "node": "<WorkerController gw2>"}
{"nodeid": "tests27_56_5.py::test_presence_of_inserted_element", "location": ["tests27_56_5.py", 4, "test_presence_of_inserted_element"], "keywords": {"test_presence_of_inserted_element": 1, "tests27_56_5.py": 1, "Folder_56": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/Command_T_D/Q27/command_results_5/Folder_56/generated_answer.py", "lineno": 7, "message": "Failed: Timeout >7.0s"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_presence_of_inserted_element():", "        initial_list = [set() for i in range((21 + 1) * 2)]", ">       returned_list = insert_after_index(initial_list)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q27/command_results_5/Folder_56/tests27_56_5.py", "lineno": 7, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def insert_after_index(nums):", "        ins_idx = 21", "        ins_val = 655.24", "        idx = sys.maxsize - 1", ">       while idx > ins_idx:", "E       Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["nums", "[set(), set(), set(), set(), set(), set(), ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q27/command_results_5/Folder_56/generated_answer.py", "lineno": 7, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_presence_of_inserted_element():", "        initial_list = [set() for i in range((21 + 1) * 2)]", ">       returned_list = insert_after_index(initial_list)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q27/command_results_5/Folder_56/tests27_56_5.py", "lineno": 7, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def insert_after_index(nums):", "        ins_idx = 21", "        ins_val = 655.24", "        idx = sys.maxsize - 1", ">       while idx > ins_idx:", "E       Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["nums", "[set(), set(), set(), set(), set(), set(), ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q27/command_results_5/Folder_56/generated_answer.py", "lineno": 7, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/Command_T_D/Q27/command_results_5/Folder_56/generated_answer.py", "lineno": 7, "message": "Failed: Timeout >7.0s"}, null]]}, "when": "call", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (138314090808896) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 6.999834088026546, "start": 1731092426.0880535, "stop": 1731092433.0878885, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "fc29ca3d780c47baae7d1c5c924dd24a", "node": "<WorkerController gw0>"}
{"nodeid": "tests27_56_5.py::test_presence_of_inserted_element", "location": ["tests27_56_5.py", 4, "test_presence_of_inserted_element"], "keywords": {"test_presence_of_inserted_element": 1, "tests27_56_5.py": 1, "Folder_56": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (138314090808896) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 0.00010339391883462667, "start": 1731092433.097247, "stop": 1731092433.097351, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "fc29ca3d780c47baae7d1c5c924dd24a", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
